test/test_sparse_csr.py

```diff
@@ -1282,6 +1289,38 @@ def test_direct_coo_csr_conversion(self, device, dtype):
 
             self.assertEqual(coo_sparse.to_sparse_csr().to_sparse_coo(), coo_sparse)
 
+    @ops(sparse_csr_linalg_solve)
```

Comment:
I understand the intention to reuse the sample inputs defined for `linalg.solve`. Is it possible to just import `sample_inputs_linalg_solve` from `common_methods_invocations.py`?

Reply:
Thanks for the question, Ivan. From what I tried, I don't think it's possible, since the `sample_inputs_linalg_solve` function takes an OpInfo object as an input, and in any case, we'll have to pass it there. Plus, the function uses helper functions from the `common_methods_invocations.py` file, so not sure if that will work. 

The signature of the sample inputs function is: `(op_info, device, dtype, requires_grad=False, vector_rhs_allowed=True, **kwargs)`.

Reply:
`op_info` is not used inside `sample_inputs_linalg_solve` so we can just pass `None`.
That way we can remove `supports_sparse_csr=True` from OpInfo of linalg.solve and remove the skip inside the consistency test.

Reply:
Done! :) Thanks, Ivan.